include/circt/Dialect/RTL/RTLStructure.td

```diff
@@ -138,6 +138,102 @@ def RTLModuleExternOp : RTLOp<"module.extern",
   let parser = "return ::parse$cppClass(parser, result);";
 }
 
```

Comment:
Please manually wrap these lines to fit in 80 columns in this file.  Clang format doesn't do tblgen :(